etc/environment-mpi4py.yml

```diff
@@ -10,6 +10,3 @@ dependencies:
   - mpich
   - mpi4py-fft>=2.0.2
   - mpi4py>=3.0.0
-  - pytest
-  - pytest-cov
-  - pytest-timeout
```

Comment:
@pancetta why was `pytest-order` missing here? Are we even using that?

Reply:
I.... don't know.

Reply:
We use it only in one project (TOMS). As the tests there are marked with "base", there is no need for pytest-order there. For simplicity I included it in environment-tests anyway, as one job needs it.

Reply:
Not sure why I did this [here](https://github.com/Parallel-in-Time/pySDC/blob/master/pySDC/tests/test_projects/test_TOMS/test_AllenCahn_contracting_circle.py), but I guess by merging the two tests we could get rid of it.

Reply:
Actually, what you did there is quite nice! I suggest we keep this module and use it more often for plotting for the website. Personally, I really like splitting the tests as much as possible so you get fine-grained feedback, but then you cannot plot all of the results in a single plot without this module.

Reply:
Fine by me!